apps/wallet/src/components/StackContainer/index.tsx

```diff
@@ -13,7 +13,7 @@ import StackHeader from './Header';
 interface Props {
 	title: string;
 	style?: StyleProp<ViewStyle>;
-	noHeader?: boolean;
+	headerActive?: boolean;
```

Comment:
should be `isHeaderActive` or you can keep it the same (`noHeader`)

Reply:
I see people use `enableHeader` or `showHeader` @ltminhthu :v

Reply:
vote for @ltminhthu suggestion